.github/workflows/tracks.yml

```diff
@@ -0,0 +1,39 @@
+name: Tracks
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  race:
+    name: Run `${{ matrix.track }}` on stack ${{ matrix.stack-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        stack-version: ["8.3.2"]
```

Comment:
this may be better as `latest` since the target is the master branch. it could be varied per target branch to match that branch's stack version

Reply:
Rally does not like `latest` as version. See https://github.com/cavokz/rally-tracks/runs/7325773069?check_suite_focus=true.

Reply:
sorry, that should be `--revision` flag now instead of `--distribution-version` if going that route: https://esrally.readthedocs.io/en/stable/command_line_reference.html?highlight=latest#revision